### DISCUSSION TITLE
v3.4 for rhel?

### DISCUSSION THREAD

**Discussion by xpseudonym** [containers/podman#12956] [Q&A] [answered]
Is there any eta for v3.4 landing in rhel?  Or alternatives to waiting - Issue #11438 is giving me untold grief...
Thanks

**Comment by rhatdan** [comment, marked as answer]
Is should be in RHEL 8.5.0.2 Which comes in February, I believe.

**Reply by xpseudonym** [reply]
Many thanks - I was worried it might have been 9

**Comment by Joerg-L** [comment]
Is there any "workaround" available to getting a Podman 3.4.x on en RHEL 8.3?
